### PR TITLE
Replace image table in dataset view with image gallery.

### PR DESCRIPTION
**Pull request by hsorby**
# Description

Replacing existing image table tab in dataset page with an image gallery with thumbnails.  Thumbnails link out to the respective view one of image or scaffold in a new tab.

## Type of change

Delete those that don't apply.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

1. Navigate to the Find Data page.
2. Click on a dataset that contains images 'Distribution of ICN Neurons in Male and Female 3D Reconstructed Rat Hearts' is a suitable choice.
3. Go to Gallery tab.
4. Thumbnail images gathered from the dataset should be visible in a strip with some contextual information.
5. Click on a thumbnail.
6. In a new tab a viewer should open for the type of thumbnail clicked. Possible outcomes are 2D image viewer, 3D image viewer, or scaffold viewer.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by cameronbaney** [comment]
@hsorby Can you please provide a test script in the "How has this been tested?" This is usually the manual steps that one would take as a user to ensure this feature is working. You can see examples of this in #150 and #154

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  trailingComma: 'all',
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
+  arrowParens: 'always',
+}
```
Comment (cameronbaney) [inline .prettierrc.js:7]: I am hesitant to modify the code styles we [already have in place](https://github.com/nih-sparc/sparc-app/blob/master/.eslintrc.js). We are already following `prettier/recommended` which has all of these except `trailingComma: 'all'`.

File: services/discover.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
+  baseURL: 'https://api.blackfynn.io/discover/datasets',
```
Comment (cameronbaney) [inline services/discover.js:4]:
```suggestion
  baseURL: `${process.env.discover_api_host}/datasets`,
```

**Comment by hsorby** [comment]
Thanks @cameronbaney I think I have addressed all comments now.

**Pull request merged**